index.html

```diff
@@ -0,0 +1,174 @@
+            <p>defer src="app.js"</p>
+            <p>(The app.js is an additional js file where you will specify how you want your slick carousels to work. you will have to create this file your self using the tools of your IDE of choice)</p>
+        </div>
+        <div>
+            <h2>Good To Go</h2>
+            <img src="images/thumbsUp.jpg" width="600px">
+            <p>Congratulations!</p>
+            <p>Your set up is now complete and we can move on to learning how to use slick</p>
+        </div>
+    </div>
+    </div>
+
+    <div>
+        <h1>How To Use Slick</h1>
+        <div class ="basic">
+            <div>
+                <h2>Creating a Slick Carousel</h2>
+                <img src="images/slick example.PNG">
+                <p>1. First you will need to create a div element and give it a class, then you will have to put and elements you want in the carousel inside that div</p>
+                <p>2. Create a "$(document).ready(function(){})" in your app.js</p>
+                <p>3. Within this function you will put $(".your-class-name").slick()</p>
+                <p>4. Within the slick() at the end of the previous step you will set the attributes of the slick carousel. (we will go over all the attributes on the next slides)</p>
+                <p>the image above is an example of what it should look like with some additional attributes</p>
+            </div>
+            <div>
+                <h2>Attributes</h2>
+                <p>Slick comes with many different ways to customize the look and functionality of its carousels.</p>
+                <p>We will start by going over the ones you saw in the image on the last slide.</p>
+            </div>
+            <div>
+                <h2>Dots and infinite</h2>
+                <img src="images/dotsExample.PNG">
+                <img src="images/dotsInfinite.PNG">
+                <p>in your slick method you can set the "dots" attribute equal to true to have dots for each slide in the carousel appear at the bottom</p>
+                <p>Likewise you can set the "infinite" attribute equal to true to have the carousel loop back to the first slide when it gets to the end</p>
+            </div>
+            <div>
+                <h2>SlidesToShow, SlidesToScroll and Speed</h2>
+                <img src="images/speedSlideShowSlideScroll.PNG">
+                <p>The SlidesToShow attribute and SlidesToScroll attribute will ditermine how many slides are displayed at once and how many slides it will scroll when the button is clicked respectively</p>
+                <p>The Speed attribute will determine how quicky the scrolling of the slides occurs</p>
+            </div>
+            <div>
+                <h2>centerMode</h2>
+                <img src="images/centermode.PNG" width="600px">
+                <br>
+                <img src="images/autoplay.PNG">
+                <p>You can use centerMode the center your carousel and change its look</p>
```

Comment:
It would have been good to describe some of the different possible modes and their effects